magma/circuit.py

```diff
@@ -232,12 +231,11 @@ def _add_intermediate_value(value):
     `CircuitKind.__repr__`.
     """
     root = value.name.root()
-    # root is either None, or a NamedRef. If it is a NamedRef which refers to a
-    # constant, then we can proceed with the empty set. Otherwise, we seed the
-    # set with the root value.
-    if root is None or root.value().const():
-        return OrderedIdentitySet()
-    return OrderedIdentitySet((root.value(),))
+    # If it is a NamedRef which doesn't refer to a constant, then we seed the
+    # set with the root value. Otherwise, we can proceed with the empty set
```

Comment:
nit
```suggestion
    # If it is a NamedRef which doesn't refer to a constant, then we seed the
    # set with the root value. Otherwise, we can proceed with the empty set.
```